src/App.vue

```diff
@@ -149,20 +148,6 @@ export default {
 
 		onResize() {
 			this.windowHeight = window.innerHeight - document.getElementById('header').clientHeight
-		},
-		newButtonAction(e) {
-			console.debug(e)
-		},
-		log(e) {
-			console.debug(e)
-		},
-		/**
-		 * Get a conversation's name.
-		 * @param {string} token The conversation's token
-		 * @returns {string} The conversation's name
-		 */
-		getConversationName(token) {
-			return this.$store.getters.conversations[token].displayName
```

Comment:
Then we should at least block failuring here when the item is not in the list atm (for whatever reason)

Reply:
yes, definetly :)